FiveCalls/FiveCalls/CallScriptViewController.swift

```diff
@@ -189,7 +190,27 @@ class CallScriptViewController : UIViewController, IssueShareable {
         present(alertController, animated: true, completion: nil)
     }
 
-    
+    func checkForNotifications() {
+        let permissions = OneSignal.getPermissionSubscriptionState()
+
+        if permissions?.permissionStatus.hasPrompted == .some(false) {
```

Comment:
Don't need the `.some(...)` here, `false` will work.

Reply:
👍